src/PIL/MpoImagePlugin.py

```diff
@@ -95,6 +98,22 @@ def _close__fp(self):
         finally:
             self.__fp = None
 
+    @staticmethod
+    def adopt(jpeg_instance, mpheader=None):
```

Comment:
At a high level I understand why you might want to give `mpheader` a default value, but at the moment, the second line of `_after_jpeg_open` would just throw a TypeError when given a `None` value. Any thoughts?

Reply:
You're right. So now `_after_jpeg_open` will use a default value too :)